### ISSUE TITLE
race condition while starting the app

### ISSUE THREAD

**Issue by BlueHotDog** [matehat/cqerl#117]
Hey, doing 
```erlang
{:ok, _} = Application.ensure_all_started(:cqerl)
{ok, Client} = :cqerl.get_client()
```
fails with :cluster_not_configured
but.. adding a timeout seems to solve it :(

**Comment by matehat** [comment]
Thanks for reporting! It was reported in #111, and is now fixed, if you use v1.0.3

**Comment by BlueHotDog** [comment]
i still see this hapenning

**Comment by matehat** [comment]
What version do you see when you type `Application.loaded_applications`?

**Comment by matehat** [comment]
Ah, I see the difference, the other ticket was for timing issue with `add_nodes` and `get_client`, you're reporting `Application.start` and `get_client`. My bad!

**Comment by matehat** [comment]
This is fixed in v1.0.4 that I just pushed, thanks for reporting!

**Comment by BlueHotDog** [comment]
confirmed! working!
